src/jdk.jdi/share/classes/com/sun/tools/jdi/VirtualMachineManagerImpl.java

```diff
@@ -128,12 +123,7 @@ protected VirtualMachineManagerImpl() {
 
             try {
                 transportService = transportServices.next();
-            } catch (ThreadDeath x) {
-                throw x;
-            } catch (Exception x) {
-                System.err.println(x);
-                continue;
-            } catch (Error x) {
+            } catch (Exception | Error x) {
```

Comment:
Another that could be just catch `Throwable`

Reply:
@plummercj  these changes are not actually part of this PR but came from the upstream PR this was dependent on.

Reply:
I think this dates from JDK 5 when pluggable JDI connectors and transports were aded. VirtualMachineManager does specify that it tolerates configuration and deployment issues with:

"At start-up time the VirtualMachineManager attempts to load and instantiate (using the no-arg constructor) each class listed in the provider configuration file. Exceptions thrown when loading or creating the Connector are caught and ignored. In other words, the start-up process continues despite of errors."

Looking at it now, I think it should be catching ServiceConfigurationError as that is what the SL's iterator hasNext/next will throw if locating, loading or instantiating a Connector fails. This seems something for an another issue of course, just strange that the Skara bots included it.